packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/choice/telephony-line-phone-accessories-choice.controller.js

```diff
@@ -0,0 +1,161 @@
+import chunk from 'lodash/chunk';
+import orderBy from 'lodash/orderBy';
+import some from 'lodash/some';
+
+import initializeBrandList from './telephony-line-phone-accessories-choice.service';
+
+import { MAX_QUANTITY } from './telephony-line-phone-accessories-choice.constant';
+
+export default class TelecomTelephonyLinePhoneAccessoriesChoiceCtrl {
+  /* @ngInject */
+  constructor($q, $translate, TucTelephonyAccessoriesOrderProcess) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.TucTelephonyAccessoriesOrderProcess = TucTelephonyAccessoriesOrderProcess;
+  }
+
+  $onInit() {
+    this.process = null;
+    this.orderTotal = null;
+
+    this.loading = {
+      init: false,
+    };
+
+    this.error = {
+      loading: null,
+    };
+
+    this.accessoryToOrder = null;
+    this.selectedAccessory = null;
+
+    this.spinnerExtremities = {
+      min: 0,
+      max: 5,
```

Comment:
```suggestion
      max: MAX_QUANTITY,
```
? 
I don't know if there is a link

Reply:
yes it is.